lib/Model/ProjectSettings.php

```diff
@@ -59,7 +59,7 @@ class ProjectSettings implements ModelInterface, ArrayAccess, \JsonSerializable
       */
     protected static $openAPITypes = [
         'links' => '\Aternos\HangarApi\Model\LinkSection[]',
-        'tags' => 'string[]',
+        'tags' => '\Aternos\HangarApi\Model\Tag[]',
```

Comment:
string[] is the correct type here but editing generated files is annoying, so just leave it, I guess